acceptance/lib/puppet/acceptance/rpm_util.rb

```diff
@@ -4,14 +4,24 @@ module RpmUtils
       # Utilities for creating a basic rpm package and using it in tests
       @@defaults = {:repo => '/tmp/rpmrepo', :pkg => 'mypkg', :publisher => 'tstpub.lan', :version => '1.0'}
 
+      def setup(agent)
+        required_packages = ['createrepo', 'rpm-build']
+        required_packages.each do |pkg|
+          unless ((on agent, "#{pkg} --version", :acceptable_exit_codes => (0..255)).exit_code == 0) then
```

Comment:
On reflection, this should perhaps be using `yum list installed #{pkg}`.  E.g. the rpm-build package doesn't install a tool called rpm-build, so `rpm-build --version` will always be an error.